SdlLogs/SdlLogs.sublime-color-scheme

```diff
@@ -0,0 +1,123 @@
+{
+    "variables": {
+        // Define variables here
+    },
+    "globals": {
+            "background":     "#22282A", 
+            "caret":      "#AEAFAD", 
```

Comment:
@dkandyba please perform some formatting of this file

Reply:
@LuxoftAKutsan  fixed in c95c56eb0fcc3c822bc445ccb58a40238b9d2ff2